vantage6/vantage6/cli/node/create_private_key.py

```diff
@@ -59,13 +59,21 @@
     default=False,
     help="Overwrite existing private key if present",
 )
+@click.option(
+    "--mfa",
+    "ask_mfa",
+    flag_value=True,
+    default=False,
+    help="Ask for MFA code",
```

Comment:
```suggestion
    help="Ask for multi-factor authentication code. Use this if MFA is enabled on the server.",
```
Making this a bit more verbose so users easily understand when to use?